src/cocoa/toga_cocoa/libs/appkit.py

```diff
@@ -7,7 +7,7 @@
 
 from rubicon.objc import *
 from toga.constants import *
```

Comment:
At column 1: (F403) 'from toga.constants import *' used; unable to detect undefined names